content/docs/details/create-proxy/structure/header/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Create a Proxy
+publishedDate: '2022-07-18T14:00:00.0Z'
+description: Creating a proxy.
+
+category: header
+---
+
+Accessing your own self-hosted solutions and 3rd party product APIs securely from Roadie is crucial in ensuring security of data and systems. Roadie provides two kinds of proxy services to handle these scenarios. 
+
+With the Proxy feature, there’s an easier way. In this tutorial, we’re going to:
+
+1. Securely store credentials in the Roadie Secrets.
+2. Configure a basic and authenticated Proxy to access a 3rd party API.
+
+As we go through this process, you’ll learn:
```

Comment:
This section just repeats the one above

Reply:
Yeah, it's the format. Usually the 2 lists are not so similar.